app/styles/global.css

```diff
@@ -243,3 +243,11 @@ dialog {
 .word-break {
   word-break: break-word;
 }
+.fc-toolbar {
```

Comment:
This is fine, but I would rather we make a css file specifically for the calendar and only load it on the calendar route. Every byte we save is valuable.

Reply:
True that! Have created a separate file for calendar styles.